Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -63,32 +80,38 @@
     outputDirRoot=outputDir,
     rnd=rnd,
 )
+
 addSeeding(
     s,
     trackingGeometry,
     field,
+    TruthSeedRanges(pt=(1.0 * u.GeV, None), eta=(-3.0, 3.0), nHits=(9, None))
+    if ttbar_pu200
+    else TruthSeedRanges(),
     geoSelectionConfigFile=oddSeedingSel,
     outputDirRoot=outputDir,
 )
+
 addCKFTracks(
     s,
     trackingGeometry,
     field,
-    CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
+    CKFPerformanceConfig(ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0, nMeasurementsMin=6),
     outputDirRoot=outputDir,
 )
+
 s.addAlgorithm(
     acts.examples.TrackSelector(
         level=acts.logging.INFO,
         inputTrackParameters="fittedTrackParameters",
         outputTrackParameters="trackparameters",
         outputTrackIndices="outputTrackIndices",
         removeNeutral=True,
-        absEtaMax=2.5,
-        loc0Max=4.0 * u.mm,  # rho max
+        absEtaMax=3.0,
         ptMin=500 * u.MeV,
```

Comment:
Sorry, there seems to be a bit of an inconsistency here. Above, we seem to be accepting that we can reconstruct above 1 GeV, but here we allow 0.5 GeV tracks in the vertices. If the vertex finder can accept lower pT tracks, then maybe we should do the same for the ITk (ie. undo one of my suggestions there).

Reply:
no that makes a lot of sense I will change it